pkg/container/container.go

```diff
@@ -413,13 +421,18 @@ func (c *Module) List(ns string) ([]pkg.ContainerID, error) {
 
 // Delete stops and remove a container
 func (c *Module) Delete(ns string, id pkg.ContainerID) error {
+	log.Info().Str("id", string(id)).Str("ns", ns).Msg("delete container")
+
 	client, err := containerd.New(c.containerd)
 	if err != nil {
 		return err
 	}
 	defer client.Close()
+
+	// log.Debug().Msg("fetching namespace")
```

Comment:
is there a reason to keep this commented logs?

Reply:
I will re-enable all debug commented.
But first we should enable